operations/configuration.md

```diff
@@ -384,40 +384,42 @@ Returns all tax environments supported by the API.
 | `Name` | string | required | Name of the taxation. |
 | `LocalName` | string | required | Local name of the taxation. |
 
-#### Tax Rates
+#### Tax rates
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Code` | string | required | Code of the tax rate. |
 | `TaxationCode` | string | required | Code of the taxation. |
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
-| `Strategy` | [Tax Rate Strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
+| `Strategy` | [Tax rate strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
 
-#### Tax Rate Strategy
+#### Tax rate strategy
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
-| `Discriminator` | [Tax Strategy Discriminator](#tax-strategy-discriminator) | required | If tax rate is flat or relative. |
-| `Value` | object | required | Based on Tax Strategy Discriminator, e.g. [Relative Rate Strategy](#relative-rate-strategy) or [Flat Rate Strategy](#flat-rate-strategy). |
+| `Discriminator` | string | [Tax rate strategy discriminator](#tax-rate-strategy-discriminator) | required | If tax rate is flat or relative. |
```

Comment:
This is incorrectly formatted:
![image](https://user-images.githubusercontent.com/435787/85548830-a19ec600-b61f-11ea-974e-3427130f236b.png)


Should be like this:
![image](https://user-images.githubusercontent.com/435787/85548765-92b81380-b61f-11ea-9dcf-2bd70b33b27f.png)